<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Calculator">
       <attribute name="title">
        <string>Expression calculator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="11" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>X range:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="expression"/>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="result">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="calculate">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLineEdit" name="domain"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Enter your expression in infix notation:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Your result:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QPushButton" name="plot">
          <property name="text">
           <string>plot graph</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLineEdit" name="EditYRange"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Provide x value:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="xvalue"/>
        </item>
        <item row="14" column="2">
         <widget class="QChartView" name="chartView"/>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Y range:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Credit calculator</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Total credit amount:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Term (months):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Interest rate:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editCreditAmount">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>80</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editTerm">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editInterestRate">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioAnnuity">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>annuity</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioDifferentiated">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>240</y>
          <width>131</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>differentiated</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>150</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Mounthly payment:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>80</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Overpayment on credit:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>110</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Total payment:</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="monthlyPayment">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>170</y>
          <width>321</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelTotalCreditOverpay">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>80</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelTotalPayment">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>110</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="creditCalculate">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>300</y>
          <width>201</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate payment</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Deposit calculator</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Deposit amount:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Deposit term (days):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Interest rate (%):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Tax rate (%):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Periodicy of payments (days):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editDepositAmount">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editDepositTerm">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>70</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editDepositPeriodicy">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>110</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editDepositInterest">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>150</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="editDepositTax">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>190</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="depositCalculate">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>591</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>370</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Accured interest:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAccuredInterest">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>370</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Tax amount:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTaxAmount">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>330</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Deposit at the end of term:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelDepositEOT">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>290</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
